packages/opentelemetry-plugin-http/src/enums/AttributeNames.ts

```diff
@@ -15,17 +15,29 @@
  */
 
 /**
- * Attributes Names according to [OpenTelemetry attributes specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md#semantic-conventions)
+ * Attributes Names according to [OpenTelemetry attributes specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#common-attributes)
  */
 export enum AttributeNames {
-  HTTP_HOSTNAME = 'http.hostname',
+  HTTP_HOST = 'http.host',
   COMPONENT = 'component',
   HTTP_METHOD = 'http.method',
-  HTTP_PATH = 'http.path',
+  HTTP_TARGET = 'http.target',
   HTTP_ROUTE = 'http.route',
```

Comment:
What is the diff between `http.target` and `http.route`?

Reply:
route is the actual route name like `"/api/v1/users/:userId"` in express. target is the full string path including args like `"/api/v1/users/1?arg=value"`

Reply:
spec says that if we can't provide "/api/v1/users/:userId", we should set it as what we put in the span name. 
`target`: `/api/v1/users/1?arg=value`
`route`: `/api/v1/users/1` if we can't set `/api/v1/users/:userId`

Reply:
The `target` actually comes from the first line of the raw HTTP according to the spec. In this example, `/docs/index.html`

```
GET /docs/index.html HTTP/1.1
Host: www.nowhere123.com
Accept: image/gif, image/jpeg, */*
Accept-Language: en-us
Accept-Encoding: gzip, deflate
User-Agent: Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1)
```